### PR TITLE
replicate pypi doc and update result

### PR DESCRIPTION
**Pull request by MXueguang**
please see results diff in MS MARCO Document Ranking
I think the result I got is too high?

### CODE REVIEW

**Comment by lintool** [comment]
Hrm... odd - can you try replicating these first:
https://github.com/castorini/anserini/blob/master/docs/experiments-msmarco-doc-leaderboard.md

There's some overall between the runs, which will help us debug.

Also, can you check if you're returning 100 hits or 1000 hits?
Also, did you create a new conda env with `pyserini==0.10.0.1` (i.e., latest)? Wondering if it's an issue of what's published on PyPI vs. what's in the current repo (i.e., dev copy).

**Comment by lintool** [comment]
Ah, here's a clue:
https://github.com/castorini/pyserini/blob/pyserini-0.10.0.0/pyserini/search/__main__.py
Hits hard coded to 1000 - https://github.com/castorini/pyserini/blob/pyserini-0.10.0.0/pyserini/search/__main__.py#L110
Even though for MS MARCO doc we only evaluate top 100

In `0.10.0.1` - the hits arg has been parameterized:
https://github.com/castorini/pyserini/blob/pyserini-0.10.0.1/pyserini/search/__main__.py

**Comment by lintool** [comment]
@MXueguang can you run the same commands on master HEAD and compare results?

**Comment by MXueguang** [comment]
> @MXueguang can you run the same commands on master HEAD and compare results?

sure

**Comment by MXueguang** [comment]
I replicated with master HEAD, which gives the results in this PR.

The results with pyserini==0.10.0.1 are identical to the document

**Comment by lintool** [comment]
So which version was causing the issue? Can you make of note of that in the docs? Update your PR then please.

File: docs/pypi-replication.md [hunk 1]
```diff
@@ -70,6 +70,36 @@ To evaluate:
...
+  E.g., `msmarco-doc` should be change to  `ms-marco-doc`. Exactly same results are expected to obtain.
+  
+- WARNING: In `pyserini==0.10.0.0`, hits are hard coded to 1000. see [here](https://github.com/castorini/pyserini/blob/pyserini-0.10.0.0/pyserini/search/__main__.py#L110).
+  Even though for MS MARCO doc we only evaluate top 100.
+  In `pyserini==0.10.0.1`, the hits are has been parameterized. see [here](https://github.com/castorini/pyserini/blob/pyserini-0.10.0.1/pyserini/search/__main__.py#L112).
+  
+CAVEAT: current HEAD of master is getting different results.
+```
+MS MARCO passage ranking task, BM25 baseline:
+map                   	all	0.1958
```
Comment (lintool) [inline docs/pypi-replication.md:89]: Wait, aren't these matching the numbers above at (a) ^^^^?

File: docs/pypi-replication.md [hunk 2]
```diff
@@ -36,8 +36,8 @@ To evaluate:
 $ wget -O jtreceval-0.0.5-jar-with-dependencies.jar https://search.maven.org/remotecontent?filepath=uk/ac/gla/dcs/terrierteam/jtreceval/0.0.5/jtreceval-0.0.5-jar-with-dependencies.jar
 $ wget https://raw.githubusercontent.com/castorini/anserini/master/src/main/resources/topics-and-qrels/qrels.msmarco-passage.dev-subset.txt
 $ java -jar jtreceval-0.0.5-jar-with-dependencies.jar -m map -c -m recall.1000 -c qrels.msmarco-passage.dev-subset.txt run.msmarco-passage.txt
-map                   	all	0.1958
```
Comment (lintool) [inline docs/pypi-replication.md]: (a) here ->

**Review by lintool — COMMENTED** [review]
Questions? Aren't the scores matching?

File: docs/pypi-replication.md (hunk 2 above)
Comment (MXueguang) [reply docs/pypi-replication.md]: I made an update but revert it back. This is the result of HEAD of master branch. I change it back to result of `pyserini==0.10.0.1`

File: docs/pypi-replication.md (hunk 1 above)
Comment (MXueguang) [reply docs/pypi-replication.md:89]: no, we should compare to the scores here, https://github.com/castorini/pyserini/blob/master/docs/pypi-replication.md

Comment (lintool) [reply docs/pypi-replication.md:89]: Oh shoot, you're right. The difference comes from different default parameters...

ARG, this is annoying, but this is fine as is - I'll circle back and redo once we publish `0.10.0.2`.

Thanks for the replication.

**Review by lintool — APPROVED** [review]
lgtm - please go ahead and merge.

**Pull request merged**